poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
+                        {{/if}}
+                    </label>
+                </div>
+            {{/if}}
+            <div class="percentage-gauge-container">
+              <div class="percentage-gauge" style="width:{{percent}}%;">
+                  <label class="poll-answer-label" for="answer-{{key}}">{{{answer}}}</label>
```

Comment:
the value of ```{{{answer}}}``` seems to include presentation markup (```<p>``` elements in the example I reviewed).  Only [phrasing content](http://www.w3.org/html/wg/drafts/html/CR/dom.html#phrasing-content-1 is allowed as a child fo ```<label>``` (```<p>```'s are not allowed).  I would just include plain text here without any other markup.